### DISCUSSION TITLE
Updated the list of community blog posts and screencasts

### DISCUSSION THREAD

**Discussion by andreasthuen** [polymind-inc/acmebot#740] [Q&A] [answered]
Hi, afaik it is not possible to create PRs for Github Wikis, so I am posting here.
Is it possible for you to add this link to the Blog-posts-and-screencasts.md file?

- [Automating certificate ordering in Azure using Kubernetes and Keyvault Acmebot](https://andreasthuen.medium.com/automating-certificate-ordering-in-azure-using-kubernetes-and-keyvault-acmebot-aea972344b13)

Regards
Andreas

**Comment by shibayan** [comment, marked as answer]
Thank you for introducing the blog. I've added a link to the blog on the relevant page of the wiki!
